mysql-test/mysql-test-run.pl

```diff
@@ -236,7 +236,7 @@ END
 
 our $opt_fast= 0;
 our $opt_force= 0;
-our $opt_mem= $ENV{'MTR_MEM'};
+our $opt_mem= $ENV{'MTR_MEM'} ? 1 : 0;
```

Comment:
while this bit isn't needed it re-enforces it is a boolean until after the command_line_setup subroutine/